### PR TITLE
Bug 1804345: handle ingress certificate signed by custom ca

### PR DESCRIPTION
**Pull request by jkhelil**

### CODE REVIEW

File: 2/contrib/s2i/run [hunk 1]
```diff
@@ -268,6 +268,11 @@ if [[ -z "${JAVA_TOOL_OPTIONS}" ]]; then
   export JAVA_TOOL_OPTIONS
 fi
 
+# update system java keystore with custom ca bundle coming from jenkins-trust-ca-bundle configmap
+if [ -s "/etc/pki/ca-trust/source/anchors" ]; then
```
Comment (akram) [inline 2/contrib/s2i/run:272]: /etc/pki/ca-trust/source/anchors is supposed to be a directory.
the -s test work with file and test zero size , then we need another test

Comment (jkhelil) [reply 2/contrib/s2i/run:272]: This a file where the ca trust configmap is mounted, and yes i am testing it exist and more than zero bytes.
You are may referecing the /etc/pki/ca-trust/source which is the directory for system wide certificates for a rhel os where it looks for certs https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/sec-shared-system-certificates

Comment (jkhelil) [reply 2/contrib/s2i/run]: fixed

**Comment by akram** [comment]
/retitle Bug 1817436: handle ingress certificate signed by custom ca

**Comment by akram** [comment]
/retitle Bug 1804345: handle ingress certificate signed by custom ca

**Pull request merged**